src/Illuminate/Support/Number.php

```diff
@@ -265,8 +265,10 @@ public static function useLocale(string $locale)
      */
     protected static function ensureIntlExtensionIsInstalled()
     {
+        $caller = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 2)[1]['function'];
+
```

Comment:
We should move this inside the conditional so we don't need to do expensive debug_backtrace work when not needed.

Reply:
Good catch.